src/Command/SendWeatherEmail.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Email\SendWeatherNotification;
+use App\Entity\Notification;
+use App\Entity\NotificationDefinition;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(name: 'app:send-email')]
```

Comment:
Może bardziej: 

```suggestion
#[AsCommand(name: 'app:send-weather-notification-email')]
```
albo coś podobnego. Chodzi o to, żeby nazwa komendy dokładniej opisywała co ona robi.